sdk/src/constants.ts

```diff
@@ -13,24 +13,28 @@ export const MULTI_FETCH_TIMEOUT = 7 * 1000;
 type GraphqlNetworks =
   | 'mainnet'
   | 'goerli'
+  | 'sepolia'
   | 'polygon'
   | 'mumbai'
-  | 'baseGoerli'
   | 'baseMainnet'
```

Comment:
Any reason for baseMainnet vs. just base @josemarinas ?

Reply:
Its the name satsuma gave to the network